lib/identity/IdentityPublicKey.js

```diff
@@ -11,8 +12,11 @@ class IdentityPublicKey {
 
     if (rawIdentityPublicKey) {
```

Comment:
`rawIdentityPublicKey` should be a `{ }` by default and we should check every property for existence before we set as we are doing in the Document model.